common/culture/traditions/00_societal_traditions.txt

```diff
@@ -682,17 +782,17 @@ tradition_vegetarianism = {
 	}
 	
 	ai_will_do = {
-		value = 75
+		value = 50
 		if = {
 			limit = {
-				NOT = {
-					scope:character.faith = {
-						has_doctrine = tenet_dharmic_pacifism
+				NOT = { 
+					scope:character = {
+						religion = { is_in_family = rf_life }
 					}
 				}
 			}
-			multiply = 0
 		}
+		multiply = 0
 	}
 }
 
```

Comment:
This is intentional as it confers `embarkation_cost_mult = -0.25` and `can_travel_along_rivers = yes`, which cultures with `heritage_gives_sea_raiding_trigger` already have (heritage: jinyu, aquatic, gilblin, azsharic, amphibian). This doesn't mean it's entirely redundant, as there are other parameters, but I believe other cultures should have access to this tradition due to cases like Zandalari, Kul Tiras, wandering pandaren, pirates and so forth. 

How about this:
```
	is_shown = {
		OR = {
			heritage_gives_sea_raiding_trigger = yes
			can_have_tradition_seafaring_trigger = yes
		}
	}
```

```
can_have_tradition_seafaring_trigger = {
	OR = {
		is_culture_or_parent_culture_trigger = { CULTURE = culture:baradin }
		is_culture_or_parent_culture_trigger = { CULTURE = culture:tirassian }
		is_culture_or_parent_culture_trigger = { CULTURE = culture:wandering_pandaren }
		is_culture_or_parent_culture_trigger = { CULTURE = culture:pirate_culture }
		has_cultural_pillar = heritage_goblin
		has_cultural_pillar = heritage_chelonian
		
		# Default WC children cultures will not inherit
		this = culture:stormreaver
		this = culture:zandalari
	}
}
```

Reply:
```
is_shown = {
	OR = {
		heritage_gives_sea_raiding_trigger = yes
		can_have_tradition_seafaring_trigger = yes
	}
}
```
The idea is nagas don't need any tradition to raid oversea and sail rivers because it's in their aquatic nature,